quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -13,7 +13,8 @@ public enum Mechanism {
             .category(OptionCategory.CACHE)
             .description("Defines the cache mechanism for high-availability. "
                     + "By default, a 'ispn' cache is used to create a cluster between multiple server nodes. "
```

Comment:
A bit lengthy and strange to add the extra `Note` at the end.

I'd suggest something along the lines of:

By default in production mode, a 'ispn' cache is used to create a cluster between multiple server nodes. By default in development mode, a `local` cache disables clustering and is intended for development and testing purposes.

Looking at the docs:

![image](https://github.com/keycloak/keycloak/assets/2271511/ca0db7b3-3e28-43f1-8ff3-e25dfda9a8c6)

It's says `ispn` is the default, but not sure how we can easily change that since there's two different defaults, maybe we just live with it as is for now

Reply:
Changed per your suggestion.

> It's says `ispn` is the default, but not sure how we can easily change that since there's two different defaults, maybe we just live with it as is for now

Yeah, doesn't seem simple enough to do. We could probably do it as a follow-up as it concerns more than just cache option.